Base/usr/share/man/man5/af.md

```diff
@@ -1,18 +1,49 @@
 ## Name
 
-af - Application File format
+af - Application File format (.af)
 
 ## Synopsis
 
 The Application Files define System Menu entries and launcher file types / protocols.
 
 ## Description
 
-.af files are human-readable and are a subset of the INI-format, have no easily detectable filemagic. These files define System Menu entries and launcher file types / protocols.
+Application files are a subset of the INI format.
+They have no easily detectable filemagic and contain application information (App group):
 
-They are stored in [`/res/apps`](../../../../res/apps).
+| key           | description                      |
+|---------------|----------------------------------|
+| Name          | name                             |
+| Executable    | executable path                  |
+| Category      | category (optional)              |
+| Description   | description (optional)           |
+| IconPath      | application icon path (optional) |
+| RunInTerminal | run in terminal flag (optional)  |
 
-## See Also
+and launcher information (Launcher group, optional):
 
+| key       | description                           |
+|-----------|---------------------------------------|
+| FileTypes | supported file types separated by ',' |
+| Protocols | protocols separated by ','            |
+
+-------------------------------------------------------
+All application files are stored in [`/res/apps`](../../../../res/apps).
+
+## Examples
+
+[`/res/apps/Calendar.af`](../../../../res/apps/Calendar.af)
+
+```ini
+[App]
+Name=Calendar
+Executable=/bin/Calendar
+Category=Utilities
+```
+
+## See also
+
+- [ini(5)](help://man/5/ini)
 - [`Userland/Services/Taskbar/main.cpp`](../../../../../Userland/Services/Taskbar/main.cpp)
-- `Launcher::load_handlers` in [`Userland/Services/LaunchServer/Launcher.cpp`](../../../../../Userland/Services/LaunchServer/Launcher.cpp).
+- `Launcher::load_handlers`
+  in [`Userland/Services/LaunchServer/Launcher.cpp`](../../../../../Userland/Services/LaunchServer/Launcher.cpp)
```

Comment:
This change seems pointless

Reply:
It's not explicitly defined anywhere that we should end the list item with a dot. Somewhere on the man pages there are dots, somewhere not. I decided to stick to one style and not put a dot.

Reply:
I'm talking about the added line break - no dot is fine

Reply:
Ah, sorry, it should be my formatter. I'll remove it.